README.md

```diff
@@ -193,7 +193,7 @@ install rig. First you add our key to your config:
 
 Then add the rig repository:
 
-    `which sudo` echo "deb http://rig.r-pkg.org/deb rig main" > /etc/apt/sources.list.d/rig.list
+    `which sudo` sh -c "echo 'deb [arch=amd64] http://rig.r-pkg.org/deb rig main' > /etc/apt/sources.list.d/rig.list"
```

Comment:
This only works on amd64 but not on other architectures.

Reply:
1. `sudo echo "..." > file` gives permission denied because the echo gets the sudo priveges not the redirect to file;
2. specifying arch is needed so apt stops warning about i386 packages not being available

Reply:
1. oh, right, I should have tested it as a regular user....
2. I fixed this at the server side

```suggestion
    `which sudo` sh -c 'echo "deb http://rig.r-pkg.org/deb rig main" > /etc/apt/sources.list.d/rig.list'
```